plugins/Denizen/scripts/restraining_order.dsc

```diff
@@ -0,0 +1,30 @@
+restraining_order:
+    type: item
+    display name: <&f>Restraining Order
+    material: paper
+    flags:
+        uuid: <util.random_uuid>
+    lore:
+        - <&7>Right click to assign to a player!
+restraining_order_handler:
```

Comment:
Teeny tiny little nitpick but I feel like there should be a new line separating the script containers. Not sure if it's part of like some official "style" yet because of how early the project is right now though. Again, teeny tiny little nitpick 😅

Reply:
done+

Reply:
Why was this change reverted?